scripts/west_commands/runners/xsdb.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2024 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+# Author: Appana Durga Kedareswara rao
+
+"""Runner for flashing with xsdb CLI, the official programming
+   utility from AMD platforms.
+"""
+import argparse
+import os
+from runners.core import ZephyrBinaryRunner, RunnerCaps, RunnerConfig
+
+class XSDBBinaryRunner(ZephyrBinaryRunner):
+    def __init__(self, cfg: RunnerConfig, config=None, bitstream=None,
+            fsbl=None):
+        super(XSDBBinaryRunner, self).__init__(cfg)
+        self.elf_file = cfg.elf_file
+        if not config:
+            cfgfile_path = os.path.join(cfg.board_dir, 'support')
+            default = os.path.join(cfgfile_path, 'xsdb.cfg')
+            board_name = os.path.basename(cfg.board_dir)
+            if 'mbv32' in board_name and bitstream is None:
+                raise ValueError('Valid bitstream is required for mbv32 platform')
+            if 'kv260_r5' in board_name and fsbl is None:
+                raise ValueError('Valid fsbl elf is required for kv260_r5 platform')
```

Comment:
A generic runner should not contain board-specific code.

Reply:
okay will remove board specific logic